### PR TITLE
Authors

### PR DESCRIPTION
**Pull request by crawfordsm**
This is the first draft of the authors list and has been reviewed by the Paper coordinators and the Astropy coordination committee.     

The authors of the paper were to be determined by making a
significant contribution to the paper as defined as a person who wrote
one of the sections in main.tex or contributed a figure to main.tex with the
exception of the coordinating committee who are listed separately.  

Any corrections can be made by pull request to the master branch once this is merged.   Private messages about this can also be sent to crawfordsm@gmail.com.

### CODE REVIEW

File: author.tex [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+
+\author[0000-0002-2958-4738]{J. Woillez}
+\affil{European Southern Observatory - Karl-Schwarzschild-Str. 2, Garching bei München, D-85748, Germany}
+
+\author{J. Xavier Prochaska}
+\affil{Astronomy \& Astrophysics, UC Santa Cruz, 1156 High St., Santa Cruz, CA 95064 USA}
+
+\author[0000-0003-2638-7648]{V. Zabalza}
+\affil{}
+
```
Comment (adrn) [inline author.tex]: de la Vega and van Kerkwijk should be moved up for alphabetical order

File: author.tex [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+\author[0000-0003-0872-7098]{A. M. Price-Whelan}
+\affil{Department of Astrophysical Sciences, Princeton University, Princeton, NJ 08544, USA}
```
Comment (adrn) [inline author.tex]: Might be overkill, but I wonder if we should define macros for each affiliation to just ensure that places that appear multiple times have the same address?

**Comment by adrn** [comment]
OK I made a few changes. I'm in the process of making a bunch of macros for the unique affiliations, but have to run for a bit and pushed up the current state. 

I also added "collaboration" blocks to identify the three author groups.

**Comment by adrn** [comment]
OK for any affiliation that appears twice, I made a macro for it to make sure spelling and address are the same. I also replaced empty affiliations with `\noaffiliation` and replaced `\affil` with `\affiliation` (`\affil` is deprecated).

I think the formatting is pretty ugly, but I can't figure out how to add a page break after the affiliations...

**Comment by crawfordsm** [comment]
Okay, two small changes to the author list

**Comment by crawfordsm** [comment]
Still waiting on one piece of feedback and then I will merge

**Comment by hamogu** [comment]
Don't worry about ugly looking lists at this point. ApJ is going to set it in a different format anyway (for once, it's going to be two column, not one). Any fine tuning of aesthetics is a waste of time at this point.

**Review by hamogu — APPROVED** [review]

**Pull request merged**